src/data/mv_polynomial/variables.lean

```diff
@@ -425,12 +449,18 @@ section degree_of
 /-! ### `degree_of` -/
 
 /-- `degree_of n p` gives the highest power of X_n that appears in `p` -/
-def degree_of (n : σ) (p : mv_polynomial σ R) : ℕ := p.degrees.count n
+def degree_of (n : σ) (p : mv_polynomial σ R) : ℕ :=
+by letI := classical.dec_eq σ; exact p.degrees.count n
+
+lemma degree_of_def [decidable_eq σ] (n : σ) (p : mv_polynomial σ R) :
+  p.degree_of n = p.degrees.count n :=
+by convert rfl
```

Comment:
Isn't `convert rfl` just the same as `congr`?
```suggestion
by congr
```

Reply:
No, this doesn't work.